sql/src/main/java/io/crate/analyze/CreateAnalyzerAnalyzedStatement.java

```diff
@@ -87,59 +89,63 @@ public void extendedAnalyzer(String name) {
     }
 
     @Nullable
-    public String extendedAnalyzerName() {
+    String extendedAnalyzerName() {
         return extendedAnalyzerName;
     }
 
-    public void tokenDefinition(String name, Settings settings) {
+    void tokenDefinition(String name, Settings settings) {
         tokenizerDefinition = new Tuple<>(name, settings);
     }
 
     @Nullable
-    public Tuple<String, Settings> tokenizerDefinition() {
+    Tuple<String, Settings> tokenizerDefinition() {
         return tokenizerDefinition;
     }
 
-    public Settings genericAnalyzerSettings() {
+    Settings genericAnalyzerSettings() {
         if (genericAnalyzerSettings == null) {
             genericAnalyzerSettings = genericAnalyzerSettingsBuilder.build();
         }
         return genericAnalyzerSettings;
     }
 
-    public Settings.Builder genericAnalyzerSettingsBuilder() {
+    Settings.Builder genericAnalyzerSettingsBuilder() {
         return genericAnalyzerSettingsBuilder;
     }
 
-    public void addTokenFilter(String name, Settings settings) {
+    void addTokenFilter(String name, Settings settings) {
         tokenFilters.put(name, settings);
     }
 
     @Nullable
-    public Map<String, Settings> tokenFilters() {
+    Map<String, Settings> tokenFilters() {
         return tokenFilters;
     }
 
-    public void addCharFilter(String name, Settings settings) {
+    void addCharFilter(String name, Settings settings) {
         charFilters.put(name, settings);
     }
 
     @Nullable
-    public Map<String, Settings> charFilters() {
+    Map<String, Settings> charFilters() {
         return charFilters;
     }
 
-    public boolean extendsCustomAnalyzer() {
+    private boolean extendsCustomAnalyzer() {
         return extendedAnalyzerName != null
                && extendedCustomAnalyzer != null
-               && extendedCustomAnalyzer.get(String.format(Locale.ENGLISH, "index.analysis.analyzer.%s.type",
-            extendedAnalyzerName)).equals("custom");
+               && extendedCustomAnalyzer.get(ANALYZER.buildSettingChildName(extendedAnalyzerName, "type"))
+                   .equals("custom");
     }
 
-    public boolean extendsBuiltInAnalyzer() {
-        return extendedAnalyzerName != null && (extendedCustomAnalyzer == null ||
-                                                !extendedCustomAnalyzer.get(String.format(Locale.ENGLISH, "index.analysis.analyzer.%s.type",
-                                                    extendedAnalyzerName)).equals("custom"));
+    private boolean extendsBuiltInAnalyzer() {
+        return extendedAnalyzerName != null
+               && (
+                   extendedCustomAnalyzer == null
+                   ||
```

Comment:
formatting seems a bit odd?

Reply:
hm personally I like it much more than before. Maybe personal preference? (intellij likes my formatting as well ;-)